src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -1,5 +1,21 @@
 @import "../../shared-imports/vars.less";
 
+// Force controls to hide during float transition to decrease CLS penalty.
+.jw-wrapper.jw-float-transition {
+    /* stylelint-disable-next-line */
+    display: none!important;
```

Comment:
Is it possible to add more specificity to avoid using `!important`?

Reply:
In a vacuum yes, however since our player can be customized in unforseen ways I want to minimize the possibility of these customizations impacting Web Vitals field data. For any stylistic change I would definitely avoid using an `important` declaration, but given the impact to CLS score caused by the transition this PR addresses I'd like to make absolutely certain these elements are hidden.

Reply:
Also, this style is only applied for a split second before the float-on-scroll transition occurs, so it has very little impact on any actual player skinning customizations.